client/components/account-status/account-fees/index.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ExpirationBar from './expiration-bar';
+import ExpirationDescription from './expiration-description';
+import { getCurrency, formatCurrencyName } from 'utils/currency';
+import {
+	formatAccountFeesDescription,
+	getCurrentFee,
+} from 'utils/account-fees';
+
+const LearnMoreLink = ( { accountFees } ) => {
+	return (
+		<p>
+			<a
+				href={
+					accountFees.discount.length
+						? 'https://woocommerce.com/terms-conditions/woocommerce-payments-promotion/'
+						: 'https://docs.woocommerce.com/document/payments/faq/fees/'
+				}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ __( 'Learn more', 'woocommerce-payments' ) }
+			</a>
+		</p>
+	);
+};
+
+const AccountFees = ( props ) => {
+	const { accountFees } = props;
+	const baseFee = accountFees.base;
+
+	const currency = getCurrency( baseFee.currency );
+	const currencyName = formatCurrencyName( baseFee.currency );
+	const currencyCode = currency?.getCurrencyConfig()?.code;
+	const feeDescription = formatAccountFeesDescription( accountFees );
+	const currentFee = getCurrentFee( accountFees );
+
+	return (
+		<>
+			{ currencyName ? `${ currencyName } ` : null }
+			{ currencyCode ? `(${ currencyCode }) ` : null }
+			{ feeDescription }
+			<ExpirationBar feeData={ currentFee } />
+			<ExpirationDescription feeData={ currentFee } />
```

Comment:
For testing:
```suggestion
			<ExpirationBar feeData={ { volume_allowance: 100, current_volume: 60 } } />
			<ExpirationDescription feeData={ { volume_allowance: 100, current_volume: 60 } } />
```

Reply:
Works well with real data too 🥳 

![image](https://user-images.githubusercontent.com/5311119/120153024-e4d1c380-c1f6-11eb-8687-1833978b96e0.png)